Toolkit/AppDeployToolkit/AppDeployToolkitConfig.xml

```diff
@@ -168,6 +170,8 @@
 		<!-- Default text displayed in the progress bar for installations. -->
 		<Progress_MessageUninstall>Uninstallation in progress. Please wait...</Progress_MessageUninstall>
 		<!-- Default text displayed in the progress bar for uninstallations. -->
+		<Progress_MessageRepair>Repairing in progress. Please wait...</Progress_MessageRepair>

```

Comment:
Should just be "Repair in progress" to be grammatically correct (I think?)

Reply:
Ok I changed it in a new commit.